.goreleaser.yml

```diff
@@ -24,6 +24,7 @@ before:
   hooks:
     - go mod tidy
     - go generate ./...
+    - make web-build
```

Comment:
This would build the frontend every time we run `goreleaser ...` which can slow down the development quite a bit. Can we conditionally run this hook based on env or command line argument?

Reply:
Addressed this concern by only updating parameter.json file if necessary. This way `make web-build` only runs when there are web changes.